lib/model/parsers/ComposeV2/Service.js

```diff
@@ -94,6 +94,13 @@ class Service extends ServiceParser {
         const accessibility = this.accessibility;
 
         const cases = {
+            'network_mode': () => {
+                if (fieldValue === 'host') {
```

Comment:
instead do: `if (fieldValue.startsWith('service:')) {`
otherwise throw error, it's safer.